bridges/snowbridge/pallets/system-v2/src/lib.rs

```diff
@@ -199,10 +190,9 @@ pub mod pallet {
 			metadata: AssetMetadata,
 			fee: u128,
 		) -> DispatchResult {
-			let origin_location = match origin.clone().into() {
-				Ok(RawOrigin::Root) => Ok(Here.into()),
-				_ => T::SiblingOrigin::ensure_origin(origin),
-			}?;
+			let origin_location = T::SiblingOrigin::ensure_origin(origin)?;
```

Comment:
This doesn't make sense. The `origin` in this context is the system-frontend pallet on AH.

You need to add an additional `location` parameter` which is original origin, passed by the system-frontend pallet when calling this function